iconsdk/utils/validation.py

```diff
@@ -27,7 +27,10 @@
 
 
 def is_password_of_keystore_file(password) -> bool:
```

Comment:
If it is useless, would be better to remove.

Reply:
Applied it.